eng/common/TestResources/New-TestResources.ps1

```diff
@@ -55,7 +55,6 @@ param (
     [string] $Location = '',
 
     [Parameter()]
-    [ValidateSet('AzureCloud', 'AzureUSGovernment', 'AzureChinaCloud')]
```

Comment:
I don't think you should get rid of these. You can add to the set, but they also help callers know which values are valid. You can type `-env` in the environment, hit tab a couple of times, and it shows the completion set. This helps usability.

Reply:
Is there a way to have a known set but allow other un-validated options as well?

Reply:
Not that I'm aware. If adding new environments regularly is enough of a concern, please disregard my suggestion here.

Reply:
Readded validation set, with Dogfood now.